config/config.go

```diff
@@ -450,7 +450,7 @@ func NewConfig() *Config {
 	flag.StringVar(&cfg.KubernetesRedisServiceName, "kubernetes-redis-service-name", "", "Sets name for redis to be used to lookup endpoints")
 	flag.IntVar(&cfg.KubernetesRedisServicePort, "kubernetes-redis-service-port", 6379, "Sets the port for redis to be used to lookup endpoints")
 	flag.StringVar(&cfg.KubernetesBackendTrafficAlgorithmString, "kubernetes-backend-traffic-algorithm", kubernetes.TrafficPredicateAlgorithm.String(), "sets the algorithm to be used for traffic splitting between backends: traffic-predicate or traffic-segment-predicate")
-	flag.StringVar(&cfg.KubernetesDefaultLoadBalancerAlgorithm, "kubernetes-default-lb-algorithm", "roundRobin", "sets the default loadbalancer algorithm to be used for traffic splitting between backends: roundrobin, random, consistentHash, powerOfRandomNChoices. default: roundRobin")
+	flag.StringVar(&cfg.KubernetesDefaultLoadBalancerAlgorithm, "kubernetes-default-lb-algorithm", kubernetes.DefaultLoadBalancerAlgorithm, "sets the default loadbalancer algorithm to be used for traffic splitting between backends: roundrobin, random, consistentHash, powerOfRandomNChoices.")
```

Comment:
roundrobin -> roundRobin

Reply:
I think description is copy-paste from KubernetesBackendTrafficAlgorithmString - the algorithm distributes requests between _endpoints_.
Lest make it `sets the default algorithm to be used for load balancing between backend endpoints`

Reply:
Please also fix Options.DefaultLoadBalancerAlgorithm godoc
https://github.com/zalando/skipper/blob/0d5031e3c382f29d51ab43621a57bb0a9e431bff/skipper.go#L277-L278

Reply:
and https://github.com/zalando/skipper/blob/0d5031e3c382f29d51ab43621a57bb0a9e431bff/dataclients/kubernetes/kube.go#L223-L224